Elevel/Persistence/Configurations/QuestionConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Elevel.Domain.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Elevel.Infrastructure.Persistence.Configurations
+{
+    public class QuestionConfiguration
+    {
+        public void Configure(EntityTypeBuilder<Question> builder)
+        {
+            builder.ToTable("Question");
+
+            builder.HasOne(x => x.Audition)
+                .WithMany(x => x.Questions)
+                .HasForeignKey(x => x.AuditionId);
```

Comment:
if Audition is not required, so please make AuditionId nullable and mark here as .IsRequired(false)

Reply:
Done